coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -1,8 +1,9 @@
-from collections import Iterable
+from collections import Iterable, namedtuple
 import os.path
 
 from coala_utils.decorators import (
     enforce_signature, generate_eq, generate_repr)
+from coala_utils.string_processing import unescape
```

Comment:
unused import
\+
the commit message: the last sentence does not fit any more :)